docs/docs-beta/docs/guides/external-systems/iomanagers.md

```diff
@@ -3,3 +3,30 @@ title: "Managing stored data with I/O Managers"
 sidebar_position: 50
 sidebar_label: "Storing data with I/O Managers"
 ---
+
+IO Managers in Dagster provide a way to separate the code that's responsible for logical data transformation from the code that's responsible for reading and writing the results.  This can help reduce boiler plate code and make it easy to swap out where your data is stored.
```

Comment:
could be worth adding a section that covers swapping out the I/O manager for a different one to cover the "swap out where your data is stored" use case